### DISCUSSION TITLE
Spa Milan pump S100201 display lights permanent

### DISCUSSION THREAD

**Discussion by Gifford47** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#629] [Q&A]
I've a bestway lazspa Milan 2021 with a pump model S100201. There are 6 wires from pump and 4 wires from display.

I've installed a blue LLC with low pass filter and an esp with latest firmware.
After power on I hear the start beep and after that the display lights permanent. Bubbles, pump and heater are functioning.
I've tried with/without low pass filter and tried several digital pin arrangements but display doesn't work as aspected.

Which setting could be the right for this setup?
@visualapproach do you have an idea?

![14-09-2023_20-01-51.jpg](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/49484063/73eceb9c-67e5-4143-aa0d-555fa9822bfa)

![14-09-2023_19-36-19.jpg](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/49484063/589f22c9-51cc-4ac5-9808-ee5e655e9e22)

![10-09-2023_12-33-18.jpg](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/49484063/1c63314b-9796-49fc-9cb6-c625041f2844)

![14-09-2023_19-49-31.jpg](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/49484063/2788dbe5-52fd-4ce1-9f17-62b0a7273c90)

**Comment by visualapproach** [comment]
Oh that's a weird hybrid. I bet you have a 4 wire system but they had a 6 wire cable over...
Treat it like a 4 wire

**Reply by Gifford47** [reply]
Ok, but how exactly:/
The pump function only with the following settings (for a 6 pin pump) but not the display
![Screenshot_20230915_182633_Chrome.jpg](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/49484063/dd3d3380-3c1c-48ec-9c96-762d9e375dc4)

All other settings don't work. Also if i don't use D5 which is originally not connected... that's confusing :/

**Comment by kallejoken** [comment]
yes OK, I have the same problem with a Helsinki model - I would be interested in a solution too.
![PXL_20230915_173610022 MP](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/72359598/3826d198-3a84-4607-b072-64cb749bdd13)

**Comment by Gifford47** [comment]
I think here is a reference for the same issue #493

**Comment by kallejoken** [comment]
> I think here is a reference for the same issue #493


I have now ordered the S100101 pump.

 The differences to the s100201 are that the s100101 does not have a WLAN module and no airjet plus.  I will then implement the regulation with the rotary control module

**Reply by Gifford47** [reply]
Had anyone tried to cut off the preinstalled WiFi module and then install the custom one?

**Reply by visualapproach** [reply]
It doesn't work with that pump unfortunately

**Reply by DobbyNator94** [reply]
I have the S100201 Airjet model with WiFi modul. Problem is, that the oriinal WiFi is now broken.. Is it possible to add the board to it? Thank you!

**Reply by visualapproach** [reply]
Not unless you replace the motherboard with a non-wifi model I'm afraid

**Reply by CarlitoSweden** [reply]
> > I think here is a reference for the same issue #493
> 
> I have now ordered the S100101 pump.
> 
> The differences to the s100201 are that the s100101 does not have a WLAN module and no airjet plus. I will then implement the regulation with the rotary control module

@kallejoken For clarification, did you:
a. replace the whole **pump** with an S100101 pump
or
b. replace your S100201 pump's **motherboard** with an S100101 motherboard

**Comment by CarlitoSweden** [comment]
I also have the Milan (S100201) and exactly the same issue as described by @Gifford47.
Is there a motherboard I could swap to, if I want to retain full functionality, but without factory wifi?

I have read through all threads, it would be nice to have a matrix somewhere with suitable motherboard "downgrades" and potential side effects.

The manual for my mine is here, it's for both S100101 and S100201, with mentioning of "AIRJET" and "AIRJET WITH Wi-Fi"
https://ns2.bestwaycorp.com/Manual/0B690E57-150F-403E-B910-8D2CAB9839D8.pdf

@kallejoken is the "AirJet Plus" simply the 2-level massage regulation that is done by software in the S100201? Didn't even know I had it. The concept "AirJet Plus" isn't even mentioned in the manual...

Is S100101 the correct swap for my S100201? Would really appreciate some feedback, I'm confused...

**Comment by CarlitoSweden** [comment]
So the solution for my S100201 Milan, was to replace both the display and the motherboard from an S100102 available as a kit from here: https://www.ebay.co.uk/itm/305404775125

From what I have understood, the S100101 and S100102 parts are interchangeable.

The S100201 display did not work with the S100102 motherboard.

Hope this helps someone out there — the Milan is pretty popular here in Sweden.
